### PR TITLE
Change local host to match foreman port 5000 on the initial setup Readme

### PR DESCRIPTION
**Pull request by mmccullar**
Hi, the initial 8 steps for the react_on_rails installation, setup, and hello_world example, it lists localhost 3000 for the server. Foreman uses localhost 5000, and I thought it would be easier for devs that may not know this if it were included in the Readme.

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/713)

### CODE REVIEW

**Comment by justin808** [comment]
@mmccullar Looks good. It's worth pointing out that foreman defaults to any PORT value set in the environment, so if you have `export PORT=3000` in your dot files or `.envrc`, then you get whatever port you set. Please make a note of that.

**Comment by mmccullar** [comment]
@justin808 - awesome! I updated the Readme with the more detailed information. If there is anything else you would like me to do, please let me know!

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   foreman start -f Procfile.dev
   ```
 
-7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world)
+7. Foreman defaults to any PORT value set in the environment. If you have ```bash export PORT=3000 ``` in your dot file or ```bash .envrc ```, so you can set whatever port you would like to use. Otherwise, foreman will use ```bash port 5000 ```, and you would visit [localhost:5000/hello_world](http://localhost:5000/hello_world)
```
Comment (justin808) [inline README.md]: Above markdown is not correct. You will use triple backticks on one line. Take a look at the other markdown examples in this file.

Comment (justin808) [reply README.md]:
```
8. Visit [localhost:5000/hello_world](http://localhost:5000/hello_world). Note, `foreman` defaults to PORT 5000 unless you set the value of PORT in your environment. For example, you can `export PORT=3000` to use the Rails default port of 3000.
```

Comment (justin808) [reply README.md]: 8. Visit [localhost:5000/hello_world](http://localhost:5000/hello_world). Note, `foreman` defaults to PORT 5000 unless you set the value of PORT in your environment. For example, you can `export PORT=3000` to use the Rails default port of 3000.

Comment (mattmccullar) [reply README.md]: Updated! Sorry for the beginner markdown mistake.

**Pull request merged**